src/microbe_stage/systems/MicrobeDeathSystem.cs

```diff
@@ -412,7 +418,7 @@ private bool HandleMicrobeDeath(ref CellProperties cellProperties, in Entity ent
 
         // Eject compounds and build costs as corpse chunks of the cell
         SpawnCorpseChunks(ref organelleContainer, compounds, spawnSystem, worldSimulation, commandRecorder,
-            position.Position, random, null, glucose);
+            position.Position, random, null, glucose, entity.Get<MicrobeSpeciesMember>().Species.IsBacteria);
```

Comment:
Same here.

Reply:
This method already takes CellProperties as a parameter so this method will be super easy to correct.

Reply:
I'll fix this up properly (there's already an accessible reference to CellProperties here).